CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-01-08
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Kubernetes 1.25 has finished rolling out in all non-sovereign regions.
```

Comment:
Perhaps call out why the delay in availability of 1.25 in some regions due to CCOA.

Reply:
CCOAs aren't to be publicly communicated, so I would just leave it as is.